wagtail/admin/templates/wagtailadmin/pages/page_listing_header.html

```diff
@@ -20,9 +20,10 @@ <h1 class="w-sr-only">
         {% if parent_page.get_latest_revision %}
             <a href="{% url 'wagtailadmin_pages:history' parent_page.id %}"
                 class="{{ nav_icon_button_classes }}"
-                data-tippy-content="{% trans 'History' %}"
-                data-tippy-offset="[0, 0]"
-                data-tippy-placement="bottom"
+                data-controller="w-tooltip"
```

Comment:
See how you have updated all these data attributes, this is great. 

A similar change just needs to be made for any other tooltips using the `data-tippy` attributes. 

I hope that's a bit clearer @subhajit20